charts/topolvm/values.yaml

```diff
@@ -101,6 +101,16 @@ node:
 #      memory: 500Mi
 #      cpu: 500m
 
+  ## Node tolerations for server scheduling to nodes with taints
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
```

Comment:
Hi

I think that it should be this ?
https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
Instead of this:
https://kubernetes.io/docs/concepts/configuration/assign-pod-node/